locales/en-US.json

```diff
@@ -76,24 +76,24 @@
   "Only show notifications (if there are any): ": "Only show notifications (if there are any): ",
   "Data preferences": "Data preferences",
   "Clear watch history": "Clear watch history",
-  "Import/Export data": "Import/Export data",
+  "Import/export data": "Import/export data",
   "Manage subscriptions": "Manage subscriptions",
   "Watch history": "Watch history",
   "Delete account": "Delete account",
   "Save preferences": "Save preferences",
   "Subscription manager": "Subscription manager",
   "`x` subscriptions": "`x` subscriptions",
-  "Import/Export": "Import/Export",
+  "Import/export": "Import/export",
```

Comment:
This is a headline?

Reply:
It appears in `Data preferences`, here:
![image](https://user-images.githubusercontent.com/14208607/52924807-ef047800-32f3-11e9-83de-b2e7c1ef905f.png)

Other locales have a similar capitalization to `Import/export`, so what's proposed here LGTM.